fineract-client/.openapi-generator-ignore

```diff
@@ -0,0 +1,11 @@
+**/README.md
+**/pom.xml
```

Comment:
Here either of **pom.xml** or **gradle** files are required for building the generated client SDK right? Or are we following any other method to build the generated SDK here? 

@vorburger @vidakovic

Reply:
I went through the **[build.gradle](https://github.com/apache/fineract/blob/fc289c94e3bcd9104195f409af27f9c7305870cb/fineract-client/build.gradle#L38)** file of **fineract-client** and figured out **buildJavaSdk**.
 
After executing this task, the SDK is generated but since pom.xml or any of the gradle build scripts are excluded, I couldn't build the generated SDK.